### PR TITLE
add meetup #10

### PR DESCRIPTION
**Pull request by hardfire**
I was thinking to change the permalinks of the meetups from /meetup/<date> to /meetup/<meetup_number> any thoughts on that ? or does it even make sense ?

### CODE REVIEW

File: _meetup/8-Mar-2014.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - name: "Max Ogden"
+    title: "Node JS Fundamentals and NPM Best Practices"
+    prezi: "https://gist.github.com/maxogden/9425694"
+  - speakers:
+      - name: "Ujjwal Ojha"
+    title: "Composer"
+    prezi: "http://www.slideshare.net/ujjwal-ojha/composer-php-dependency-manager"
+  - speakers:
+      - name: "Rhoitman Amatya"
+    title: "Git
```
Comment (geshan) [inline _meetup/8-Mar-2014.md]: missing " add it please

Comment (hardfire) [inline _meetup/8-Mar-2014.md]: oh yes! thanks. that's why we have pull requests :D

**Comment by hardfire** [comment]
@geshan  this one ? ready to merge as well

**Comment by geshan** [comment]
its fine for now, for later squash the 2 commit to 1 :)

**Comment by hardfire** [comment]
ah! true. will do that. github's web interface doesnt have the option.

**Pull request merged**